### PR TITLE
chore(cache): Update TTLs / docs

### PR DESCRIPTION
**Pull request by damassi**
The type of this PR is: **Chore**

### Description

In an effort to improve our cache hit rate, this bumps our server-side TTLs for a number of routes via a new `Apps/serverCacheTTLs.tsx` index file. Most of these choices I believe are uncontroversial but let me know if anything should be tweaked, or if I missed a route that should be bumped up more appropriately!

### CODE REVIEW

File: src/Apps/serverCacheTTLs.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  artist: HOURS_24,
+  artists: HOURS_24,
+  artistSeries: HOURS_24,
+  auction: NO_CACHE,
+  categories: HOURS_24,
+  collect: HOURS_24,
+  collection: HOURS_24,
+  collections: HOURS_24,
+  gene: HOURS_24,
+  partners: HOURS_24,
```
Comment (starsirius) [inline src/Apps/serverCacheTTLs.tsx:19]: Some thoughts:

- Partners pay attentions to their listing pages (e.g. partners, partner) and I assume we'd get questions from partners if updates take max 1 day to be reflected on the site. _I think_ it's a relatively lower traffic route, so maybe it's safer to use a shorter/more reasonable TTL. Also, it's worth giving the galleries team a heads-up about the caching behavior so they can be prepared [when getting questions](https://artsy.slack.com/archives/C03N12SR0RK/p1722867890398409?thread_ts=1722861744.869819&cid=C03N12SR0RK).
- Not specific to this route, but for timely information (such as the urgency signals that Emerald is working on), updates would take max 1 day to be reflected. Not sure how significant the impact is to conversion, but I'll surface to our team to at least be aware of it.

File: docs/caching.md [hunk 2]
```diff
@@ -5,11 +5,21 @@ There are a few different layers of caching on Artsy.net, namely:
...
 > ```
 > hokusai production env unset ENABLE_GRAPHQL_PROXY ENABLE_GRAPHQL_CACHE
 > ```
 
+#### Important Cache-related ENV vars:
+
+- `ENABLE_GRAPHQL_PROXY`: Enables the proxy in support of caching
+- `ENABLE_GRAPHQL_CACHE`: Enables the GraphQL cache
+- `GRAPHQL_CACHE_SIZE`: Total number of entries before eviction
+- `GRAPHQL_CACHE_SIZE`: Default expiration TTL
```
Comment (starsirius) [inline docs/caching.md:19]: It's not immediately clear to me about the unit, so maybe we can say it's in milliseconds.

File: src/Apps/serverCacheTTLs.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/serverCacheTTLs.tsx:19]: Once the numbers stabilize we'll make a wider announcement! PDDE Core has already been informed and we as engineers should keep an eye on things. We're still getting a very low % cache hit rate.

Comment (damassi) [reply src/Apps/serverCacheTTLs.tsx:19]: But yes, that's a good point regarding partners being lower traffic, higher visibility (and not necessarily from collectors). I'll remove this entry so as to reduce the potential for confusion 👍

**Review by joeyAghion — APPROVED** [review]

**Comment by sweir27** [comment]
Sorry for the late review – this looks reasonable to me and thank you for the clear config file! It's great to see it all in one place – makes it easier to reason about.

**Pull request merged**